qprofiler2/test/org/qcmg/qprofiler2/summarise/KmersSummaryTest.java

```diff
@@ -158,7 +156,7 @@ public void toXmlFastqTest() throws ParserConfigurationException {
 	}
 
 	@Test
-	public void toXmlTest() throws IOException, DOMException, ParserConfigurationException {		
+	public void toXmlTest() throws IllegalArgumentException, ParserConfigurationException, IOException {		
```

Comment:
IllegalArgumentException is an unchecked exception and so you don't need it in the throws clause